### ISSUE TITLE
Add example of "pp subito" to official examples

### ISSUE THREAD

**Issue by mscuthbert** [w3c/musicxml#509]
From @mdgood 's comment at #504 -- proper encoding is:

```xml
<direction placement="below">
    <direction-type>
        <dynamics>
            <pp/>
        </dynamics>
    </direction-type>
    <direction-type>
        <words font-style="italic"> subito</words>
    </direction-type>
    <sound dynamics="36"/>
</direction>
```

**Comment by lemzwerg** [comment]
Hmm, shouldn't this be rather

```xml
<words xml:space="yes" font-style="italic"> subito</words>
```

since the default of the `xml:space` attribute is 'no'?

**Comment by mdgood** [comment]
No, that would be invalid XML. The allowed values for xml:space are "default" and "preserve" and the attribute does not have a value if unspecified. One could add the `xml:space="preserve"` attribute.

**Comment by lemzwerg** [comment]
Ah, my mistake, sorry.  I've meant `preserve`. 

But wouldn't your change from `subito` to ` subito` have no effect if `xml:space`  weren't set to `preserve`?

**Comment by lemzwerg** [comment]
Or maybe even better: Explain in the documentation that whitespace between `<words>` and non-`<words>` elements is implementation-defined.  While I'm not a friend of 'implementation-defined' stuff in a specification in general I don't see a good solution to this particular conundrum, because a dynamics 'p' is certainly not a normal text object – this makes it nearly impossible to define clean rules for interaction with text.  For example, a `<p/>` glyph might provide much more horizontal padding around it to (a) avoid collisions with other objects, and (b) to attach words like 'subito' directly, not needing to take spacing into account.

**Comment by lemzwerg** [comment]
Please forget my remarks.  I misunderstood what `xml:space="preserve"` does.  Sorry for the noise.

**Comment by lemzwerg** [comment]
My comments above are not as noisy as I believed then :slightly_smiling_face: 

I think that the example should really be changed to

```xml
<direction placement="below">
    <direction-type>
        <dynamics>
            <pp/>
        </dynamics>
    </direction-type>
    <direction-type>
        <words font-style="italic" xml:space="preserve"> subito</words>
    </direction-type>
    <sound dynamics="36"/>
</direction>
```

as @mdgood suggested.

My conclusion from the lengthy discussion of #571 is that the recommended XML behaviour to handle contents of elements is to strip off leading and trailing whitespace, convert all whitespace to normal space characters, and condense a sequence of space characters to a single space.  This means that only `xml:space="preserve"` guarantees that the leading space in the `<words>` field survive.  Otherwise we are at the mercy of the implementation-specific whitespace handling.
